src/room007/models/model.py

```diff
@@ -38,6 +44,43 @@ def option_combination_name(options):
         else:
             return [('||| no parameter combinations |||', {option_name: option.default for option_name, option in self.options.items()})]
 
+
+class FeatureManager(object):
+    save = False
```

Comment:
Fair enough. I still don't like the mutable class field but also I couldn't think of a comparably easy solution to separating the logic of managing features. It's certainly good to have it concentrated in one place.